SingleSource/Benchmarks/Polybench/README

```diff
@@ -1,9 +1,145 @@
+
+Contact:
+   Louis-Noel Pouchet <pouchet@cse.ohio-state.edu>
+   Tomofumi Yuki <tomofumi.yuki@inria.fr>
+
+
+PolyBench is a benchmark suite of 30 numerical computations with
+static control flow, extracted from operations in various application
+domains (linear algebra computations, image processing, physics
+simulation, dynamic programming, statistics, etc.). PolyBench features
+include:
+- A single file, tunable at compile-time, used for the kernel
+  instrumentation. It performs extra operations such as cache flushing
+  before the kernel execution, and can set real-time scheduling to
+  prevent OS interference.
+- Non-null data initialization, and live-out data dump.
+- Syntactic constructs to prevent any dead code elimination on the kernel.
+- Parametric loop bounds in the kernels, for general-purpose implementation.
+- Clear kernel marking, using pragma-based delimiters.
+
+
+PolyBench is currently available in C and in Fortran:
+- See PolyBench/C 4.2.1 for the C version
+- See PolyBench/Fortran 1.0 for the Fortran version (based on PolyBench/C 3.2)
+
+Available benchmarks (PolyBench/C 4.2.1)
+
+Benchmark	Description
+2mm		2 Matrix Multiplications (alpha * A * B * C + beta * D)
+3mm		3 Matrix Multiplications ((A*B)*(C*D))
+adi		Alternating Direction Implicit solver
+atax		Matrix Transpose and Vector Multiplication
+bicg		BiCG Sub Kernel of BiCGStab Linear Solver
+cholesky	Cholesky Decomposition
+correlation	Correlation Computation
+covariance	Covariance Computation
+deriche		Edge detection filter
+doitgen		Multi-resolution analysis kernel (MADNESS)
+durbin		Toeplitz system solver
+fdtd-2d		2-D Finite Different Time Domain Kernel
+gemm		Matrix-multiply C=alpha.A.B+beta.C
+gemver		Vector Multiplication and Matrix Addition
+gesummv		Scalar, Vector and Matrix Multiplication
+gramschmidt	Gram-Schmidt decomposition
+head-3d		Heat equation over 3D data domain
+jacobi-1D	1-D Jacobi stencil computation
+jacobi-2D	2-D Jacobi stencil computation
+lu		LU decomposition
+ludcmp		LU decomposition followed by Forward Substitution
+mvt		Matrix Vector Product and Transpose
+nussinov	Dynamic programming algorithm for sequence alignment
+seidel		2-D Seidel stencil computation
+symm		Symmetric matrix-multiply
+syr2k		Symmetric rank-2k update
+syrk		Symmetric rank-k update
+trisolv		Triangular solver
+trmm		Triangular matrix-multiply
+
+
+See the end of the README for mailing lists, instructions to use
+PolyBench, etc.
+
+--------------------
+* New in 4.2.1-beta:
+--------------------
+ - Fix a bug in PAPI support, introduced in 4.2
+ - Support PAPI 5.4.x
 
-Copyright (c) 2011-2012 the Ohio State University.
-Contact: Louis-Noel Pouchet <pouchet@cse.ohio-state.edu>
+-------------
+* New in 4.2:
+-------------
+ - Fixed a bug in syr2k.
+ - Changed the data initialization function of several benchmarks.
+ - Minor updates in the documentation and PolyBench API.
+
+-------------
+* New in 4.1:
+-------------
+ - Added LICENSE.txt
```

Comment:
It looks like `LICENSE.txt` has not be included? There's `SingleSource/Benchmarks/Polybench//LICENSE`, but that still mentions 3.2 and needs updating I think

Reply:
Oops, I missed it. I just updated.